src/com/projectkorra/projectkorra/GeneralMethods.java

```diff
@@ -2030,8 +2030,13 @@ public static FallingBlock spawnFallingBlock(Location loc, Material type, byte d
 	}
 
 	public static void sendBrandingMessage(CommandSender sender, String message) {
-		ChatColor color = ChatColor.valueOf(ConfigManager.languageConfig.get().getString("Chat.Branding.Color").toUpperCase());
-		color = color == null ? ChatColor.GOLD : color;
+		ChatColor color;

```

Comment:
Needs a space after the try and catch statement to follow our formatting standard of `try {` and `catch (Exception e) {`.